src/security/credentials.rs

```diff
@@ -171,8 +231,27 @@ impl ChannelCredentialsBuilder {
             |(cert, key)| (cert.into_raw(), key.into_raw()),
         );
 
-        let creds =
-            unsafe { grpc_sys::grpcwrap_ssl_credentials_create(root_ptr, cert_ptr, key_ptr) };
+        let mut pair = grpcio_sys::grpc_ssl_pem_key_cert_pair {
+            private_key: key_ptr,
+            cert_chain: cert_ptr,
+        };
+        let creds = unsafe {
+            if cert_ptr.is_null() {
```

Comment:
Can you explain why want to add this judgment? I think if the user does not input correctly, it should panic instead of not passing in null by default.

Reply:
It's a valid usage allowed by grpc.

Reply:
What about returning an error instead?

Reply:
It's a valid usage that should not report error or panic. The statement "user does not input correctly" is just wrong.

Reply:
Yeah, just root cert is OK, but incomplete cert_chain is not ok.